features/concurrent-user-sharing.feature

```diff
@@ -0,0 +1,18 @@
+Feature: concurrent user sharing
+  As a user
+  I want to share their resource concurrently to different multiple users
+
+  @concurrent
```

Comment:
Why do we need this tag? To quickly run the `go test --godog.tags=concurrent` test or do we need it in the future?

Reply:
oh sorry i forgot to remove it as this was part of local testing. Removing it.